packages/app/frontend/src/main.tsx

```diff
@@ -20,6 +20,7 @@ const localeSwitch = (lang: string) =>
     .with('en-US.UTF-8', () => locales.enUS)
     .with('de-DE.UTF-8', () => locales.deDE)
     .with('vi-VN.UTF-8', () => locales.viVN)
+    .with('ES-SPA.UTF-8', () => locales.viVN)
```

Comment:
I think locale could would be `es_ES.UTF-8`, should probably also be renamed in the other files for consistency
Also it returns Vietnamese instead of Spanish here :P

Reply:
oops my bad! lemme fix that

Reply:
all fixed!